doc/Makefile

```diff
@@ -134,10 +134,16 @@ pdf: latex
 
 install:
 	rm -rf _build/doctrees/
-	make html
+	# first clone the nilearn.github.io repo because it may ask
+	# for password and we don't want to delay this long build in
+	# the middle of it
 	rm -rf _build/nilearn.github.io
+	# --no-checkout just fetches the root folder without content
+	# --depth 1 is a speed optimization since we don't need the
+	# --history prior to the last commit
```

Comment:
Arghh my editor was trying to be helpful when reformating the paragraph and I thought I removed that. Oh well, done now.